### DISCUSSION TITLE
Line numbering in CodeBlock

### DISCUSSION THREAD

**Discussion by tibotiber** [facebook/docusaurus#5988] [Ideas]
Hi, it would be great to support line numbering in CodeBlock. Any plans to do that, or is it already available but I couldn't find? Thanks.

**Comment by Josh-Cena** [comment]
It's not currently supported, but you can achieve that in userland through [swizzling](https://docusaurus.io/docs/next/using-themes#swizzling-theme-components). Just add a `<span>` containing the line number to every code line.

We may consider adding that feature, after working on #5783

**Reply by tibotiber** [reply]
Thanks for the quick reply @Josh-Cena. Looking forward to the potential addition.
